data/state/peerAccount.go

```diff
@@ -321,6 +321,20 @@ func (pa *PeerAccount) IncreaseValidatorSuccessRateWithJournal(value uint32) err
 	return pa.accountTracker.SaveAccount(pa)
 }
 
+// UpdateValidatorSuccessRateWithJournal increases the account's number of successful signing with the provided amount,
+// saving the old state before changing
+func (pa *PeerAccount) UpdateValidatorSuccessRateWithJournal(amount uint32) error {
```

Comment:
the new methods are exactly the same as the changed methods with value. Is it really needed?

Reply:
removed